packages/strapi-utils/lib/models.js

```diff
@@ -75,6 +75,13 @@ module.exports = {
         });
       });
     } else if (_.has(attribute, 'via') && _.has(attribute, 'collection')) {
+      if (!_.has(models, attribute.collection)) {
+        throw new Error(
+          `The collection \`${_.upperFirst(
+            attribute.collection
+          )}\` is missing from the models ${attribute.plugin ? '(plugin - ' + attribute.plugin + ')' : ''}`
```

Comment:
to make the sentence clearer I would write `the collection x is missing from the plugin x models.`

Reply:
Thank you for the feedback, I'm gonna update it !